### ISSUE TITLE
Print invocation url to stdout after building with BES enabled

### ISSUE THREAD

**Issue by sgreenstein** [bazelbuild/bazel#4143]
### Description of the problem / feature request / question:

When I build with BES enabled, it would be very convenient to have a link to the uploaded invocation printed to the *end* of the stdout. e.g. the last line of the build logs should be https://source.cloud.google.com/results/invocation/c257a6e1-cad8-9d30-7808-ca5d65a12d53

### If possible, provide a minimal example to reproduce the problem:
```
sgreens@sgreens0$ bazel build //libs/... --bes_backend="buildeventservice.googleapis.com" --auth_credentials=... --bes_best_effort=false --bes_timeout=10s --tls_enabled --auth_enabled --auth_scope=https://www.googleapis.com/auth/cloud-source-tools --project_id=...
INFO: Streaming Build Event Protocol to buildeventservice.googleapis.com build_request_id: 53c444ca-a532-31c2-cada-2c4189a40aea invocation_id: c257a6e1-cad8-9d30-7808-ca5d65a12d53
...
tons of build logs
...
```

I then have to scroll to the top of the build output, copy the invocation_id, and prepend part of the url to get this:
https://source.cloud.google.com/results/invocation/c257a6e1-cad8-9d30-7808-ca5d65a12d53

### Environment info
* Operating System: Linux

* Bazel version (output of `bazel info release`): 0.6.1

###  Have you found anything relevant by searching the web?
No

**Comment by meteorcloudy** [comment]
//cc @aehlig @buchgr

**Comment by sgreenstein** [comment]
Possible dupe of #4628

**Comment by jin** [comment]
@sgreenstein yep, this is a dupe.

**Comment by sgreenstein** [comment]
Turns out it's not a dupe: I just tried out --bes_results_url and it makes things much more convenient (not having to type out the url prefix); however, it still prints it before all the build logs and not after.

There's a big difference in usability: printing it at the beginning means you have a half second to click on the link or scroll up through all the build logs to the right place afterward. Printing it after the build completes is much more usable

**Comment by sgreenstein** [comment]
This appears to be fixed in 0.16.0
